new/LIT7102QeneYom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7102QeneYom" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ዮም፡ ዕለተ፡ ብዙኅ፡ ትፍሥሕት፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Yom ʿəlata bəzuḫ təfśəḥt (Qəne of tha gubāʾe qānā-type)</title>
+                <title xml:lang="en" corresp="#t1">Today is a day of much joy…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="JK"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Gubāʾa Qānā</foreign>-type <foreign xml:lang="gez">qəne</foreign>. It has been
+                    published in <bibl><ptr target="bm:Afawarq1984Yageezqeneyat"></ptr><citedRange unit="page">230-231, no. 246</citedRange></bibl>,
+                    where it is attributed to <persName ref="PRS14578KeflaYohannes" role="author"/>. It also appears as part of 
```

Comment:
According to the edition, it is chanted in `<ref type="authFile" corresp="EzelMode">ʿǝzǝl mode</ref>`